nodes/config/ui_base.html

```diff
@@ -1038,7 +1042,12 @@
             addItem: function (container, i, /** @type {DashboardItem} */ widget) {
                 const titleRow = $('<div>', { class: 'nrdb2-sb-list-header nrdb2-sb-widgets-list-header' }).appendTo(container)
                 $('<i class="nrdb2-sb-list-handle nrdb2-sb-widget-list-handle fa fa-bars"></i>').appendTo(titleRow)
-                let widgetIcon = 'fa fa-image'
+
+                let widgetIcon = widget.node._def.icon || 'fa fa-question'
+                if (widget.node._def.icon.startsWith('font-awesome/')) {
```

Comment:
This breaks the sidebar. subflows icon can be a function.

![image](https://github.com/user-attachments/assets/a297a221-b57b-4624-99ad-f17bcce87a4f)